test/fastmath.jl

```diff
@@ -144,6 +144,7 @@ end
 for T in (Complex64, Complex128, Complex{BigFloat})
     half = (1+1im)/T(2)
     third = (1-1im)/T(3)
+    println("$T")
```

Comment:
It appears that `println` makes it go away, so I'm not sure what the deal is. The log is here:
https://ci.appveyor.com/project/JuliaLang/julia/build/1.0.5816/job/2lbl9vhx3yfo6x5i
(it only appeared on 64-bit Windows).